backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -267,19 +267,11 @@ fieldset {
       }
     }
 
-    span.or {
-      background-color: $color-1;
-      border-width: 5px;
-      margin-left: 5px;
-      margin-right: 11px;
-      position: relative;
-      z-index: 1;
-
-      -webkit-box-shadow: 0 0 0 5px $color-1;
-         -moz-box-shadow: 0 0 0 5px $color-1;
-          -ms-box-shadow: 0 0 0 5px $color-1;
-           -o-box-shadow: 0 0 0 5px $color-1;
-              box-shadow: 0 0 0 5px $color-1;
+    // Always make sure the choices at the bottom of a fieldset are spaced out
+    form + form,
+    button + a,
+    form + a {
```

Comment:
I went through and spot checked.  Those rules cover all existing cases and `* + *` would add no new ones.  The other occasions that did not include the `span class="or"` element are just for spots with individual buttons or links such as the search forms.